hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -601,7 +601,7 @@ ppFailureReport name (FailureReport size seed _ inputs0 mlocation0 msg mdiff msg
         WL.vsep . WL.punctuate WL.line
     , with decls $
         WL.vsep . WL.punctuate WL.line . fmap ppDeclaration
-    , with msgs $
+    , with msgs
```

Comment:
Hmm, I might be crazy, but despite being redundant I kind of prefer keeping this one because it makes all the `with` calls consistent